src/main/java/com/parasoft/report/transformer/XMLToSarif.java

```diff
@@ -181,20 +181,23 @@ private void logUnconvertedPathsInSarifReport() {
                     run.get("artifacts").elements().forEachRemaining(artifact -> {
                         if (artifact.size() > 0) {
                             JsonNode locationNode = artifact.get("location");
-                            String uri = locationNode.get("uri").asText();
                             if (!locationNode.has("uriBaseId")) {
+                                String uri = locationNode.get("uri").asText();
                                 uriSet.add(uri);
                             }
                         }
                     });
                 });
-                List<String> uriList = new ArrayList<>(uriSet);
-                Collections.sort(uriList);
-                for (String uri : uriList) {
-                    Logger.info(MessageFormat.format("The unconverted paths in the SARIF report: {0}", uri));
+                if (!uriSet.isEmpty()) {
+                    List<String> uriList = new ArrayList<>(uriSet);
+                    Collections.sort(uriList);
+                    Logger.info("The following paths have not been converted to relative paths:");
+                    for (String uri : uriList) {
+                        Logger.info(MessageFormat.format("  {0}", uri));
+                    }
                 }
             } catch (IOException e) {
-                throw new RuntimeException(MessageFormat.format("Read SARIF report error: {0}", e.getMessage()));
+                Logger.error(MessageFormat.format("Read SARIF report error: {0}", e.getMessage()));
```

Comment:
Please add "ERROR:" at the beginning of the message to be consistent with the other errors in project. 
In that